net-p2p/i2p/i2p-0.9.26.ebuild

```diff
@@ -72,15 +75,16 @@ src_prepare() {
 	if ! use nls; then
 		echo "require.gettext=false" >> override.properties
 	fi
+	eapply_user
 }
 
 src_install() {
+	# add libs
+	epatch "${FILESDIR}/${P}-add_libs.patch"
```

Comment:
don't mix patching into different phase functions. Patching should happen in `src_prepare`, preferably by adding the patches into the global `PATCHES` array variable.